client/my-sites/checkout/checkout-thank-you/licensing-thank-you-manual-activation-license-key.tsx

```diff
@@ -38,7 +38,7 @@ const LicensingActivationInstructions: FC< JetpackLicenseKeyProps > = ( {
 			/>
 			<LicensingActivation
 				title={ translate( 'Activate your %(productName)s plan', {
-					args: { productName },
+					args: { productName: productName ?? '' },
```

Comment:
If `productName ` is nullish, the string will end up having just double space. Anything we can address separately, @rcanepa  or @Automattic/jetpack-martech?

Reply:
If this component really should work if `hasProductInfo` is false, I would recommend using that to create another message potentially